src/style.css

```diff
@@ -1,3 +1,88 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./font/Roboto-Medium.ttf);
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: gray;
+  display: flex;
+  justify-items: center;
+  font-size: 12px;
+}
+
+header {
+  width: 100%;
+  background-color: rgba(255, 255, 255, 1);
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo img {
+  height: 40px;
+  width: 40px;
+}
+
+nav ul {
+  list-style: none;
+  text-align: center;
+  display: flex;
+  height: 60px;
+}
+
+nav ul li {
+  position: relative;
+}
+
+nav ul li a {
```

Comment:
Sure, we can do it like that. But better - and more valid approach - will be to use class names. 
For example: 

```
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```

Reply:
dodałem nowe classy i zmieniłem ścieżki w css.

Reply:
<img width="1792" alt="li" src="https://github.com/mate-academy/layout_moyo-header/assets/57567908/85656ec3-2d29-4255-b5e6-7aab27230535">

We should add class to each li item :)